front-end/packages/help/getting-started.md

```diff
@@ -1,26 +1,114 @@
+### Language
 
-Morbi sed velit non ipsum tempor maximus. Nulla elementum elementum magna, cursus rutrum lorem blandit in. Mauris et massa molestie, fermentum lectus mattis, sodales orci. Phasellus condimentum, purus vel aliquam facilisis, leo velit fermentum dui, eget ultricies risus mauris sit amet leo. Donec hendrerit porta turpis, ut fermentum odio ultrices laoreet. Donec volutpat ultricies sapien, at auctor ligula molestie quis. Maecenas lobortis augue at congue finibus. Aenean malesuada quis orci ac ornare. Donec efficitur dolor vitae mauris auctor cursus. Morbi sed ante sit amet urna vehicula scelerisque quis a erat. Integer imperdiet ipsum at accumsan bibendum. Nunc ultrices non tellus eget aliquam. Duis a sapien vel ligula vulputate lacinia eu eleifend purus. Curabitur vehicula euismod sem quis eleifend.
+Language section shows which languages are supported in your installation. Language checkboxes select languages for text generation.
 
-Vestibulum vitae odio vel odio gravida tincidunt. Phasellus vehicula, ex sed vehicula semper, arcu turpis tincidunt lacus, quis egestas augue lectus non tellus. Maecenas est nunc, semper eget tellus nec, congue dictum libero. Nunc sit amet tellus maximus, mattis est eget, scelerisque felis. Mauris dapibus eros nec ipsum venenatis facilisis. Maecenas interdum auctor neque, accumsan dapibus erat. Vivamus vitae ullamcorper quam. Nulla blandit sapien id eleifend blandit. Curabitur id volutpat enim. Nunc tellus mauris, blandit nec fringilla vitae, sollicitudin in ipsum. Cras viverra sapien felis, et lobortis diam luctus at. Quisque et dui id turpis finibus accumsan sed eu dolor.
+![data](help/language.png)
 
-Aliquam maximus augue a augue auctor tempor. Suspendisse sit amet dolor consectetur, bibendum orci sit amet, ornare elit. Etiam mauris diam, iaculis eu nunc non, mattis molestie nunc. Vivamus tempus volutpat augue at aliquet. Suspendisse potenti. Pellentesque vestibulum commodo tincidunt. Ut vel est suscipit ex dignissim imperdiet. Vivamus rhoncus eros metus, ut ornare arcu semper a. Morbi id pretium turpis. Sed sed nisi sed tortor placerat pulvinar. Quisque cursus felis ex, et vestibulum lacus sodales a. Nunc non finibus metus. Ut gravida, enim sed laoreet commodo, nisl turpis lobortis ante, iaculis dictum sapien nibh eget nibh. Phasellus accumsan tempor mi a gravida. Vivamus imperdiet dictum nibh, nec pulvinar mi rhoncus in. Maecenas molestie posuere blandit.
+### Text Generation Results
 
+This section will show the resulting texts. Each block of text represents variations of the text given the same document plan and current row of data. A flag next to text shows the language of the text.
```

Comment:
"block of text" -> "block", "variations of the text" -> "variations of text", "the language of the text" -> "the language"